### PR TITLE
PR for activity add Contact section to mobile project

### PR DESCRIPTION
**Pull request by DarioAlessioR**
PR to ask for approval to marge branch profolioContact into the master branch.

Branch porfolioContact contains the following changes:
- Add form for name, last name, email, and message.
- Add social media icons.
- Add email at end of the page

Features or functionalities added:
- Name field accepts up to 30 chars.
- Email field accepts only emails.
- The message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).
- Use Flexbox.

Changes done while pair programming with https://github.com/Marlyn-Mayienga

Thank you in advance for review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -187,10 +183,42 @@ <h4>About myself</h4>
...
+
+      <h4>CONTACT US</h4>
+      <p>If you have an application you are interested in developing, a feature that
+        you need built or a project that needs coding, I'd love to help with it.
+      </p>
+      
+      <div class="contact">
+        <form action="https://formspree.io/f/xrgjojpj" method="post">
+          <input type="text" id="name" name="user_name" placeholder="Name" required minlength="1" maxlength="30">
+          <input type="text" id="lastname" name="user_last_name" placeholder="Last name" required minlength="1" maxlength="30">
```
Comment (thecodechaser) [inline index.html]: - [ ] Please make sure to use only one input field for the name.

Comment (thecodechaser) [inline index.html]: - [ ] Please use correct paddings for the input fields and make sure it matches the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design.

### Your design

image

### Figma design

image

File: index.html [hunk 2]
```diff
@@ -187,10 +183,42 @@ <h4>About myself</h4>
...
+          <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+          <textarea id="msg" name="user_message" placeholder="Your message" required minlength="1" maxlength="500"></textarea>
+          
+            <button class="superButton" type="submit">Send</button>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please increase padding for the submit button and make sure it matches with the design.

### Your design

image

### Figma design

image

File: index.html [hunk 3]
```diff
@@ -187,10 +183,42 @@ <h4>About myself</h4>
...
+        </form>
+      </div>
+
+      <hr>
+      <div class="container-WorkSocial">
+        <ul>
+          <li><a href="#" target="new"><i class="fab fa-github social-icon"></i></a></li>
+          <li><a href="#" target="new"><i class="fab fa-linkedin-in social-icon"></i></a></li>
+          <li><a href="#" target="new"><i class="fab fa-facebook social-icon"></i></a></li>
+          <li><a href="#" target="new"><i class="fab fa-twitter social-icon"></i></a></li>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please reduce the gap between social media icons and make sure it matches the design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @DarioAlessioR 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
- [ ] Please make sure that every member from your coding partner group has contributed to this project, I can only see your commits, I am requesting some changes Implement the changes with your group in pair programming we will check the commit history.
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DarioAlessioR — COMMENTED** [review]
Changes commited by Marlyn approved

**Comment by DarioAlessioR** [comment]
PR ready after required changes were done. 

Project ready to ask for a new review.

Changes were done with coding partner @Marlyn-Mayienga 

Thank you in advance.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DarioAlessioR** [comment]
Project ready to merge after approval is received from the reviewer.

**Pull request merged**